src/pages/lessons/[slug].tsx

```diff
@@ -393,9 +408,14 @@ const Lesson: React.FC<LessonProps> = ({initialLesson}) => {
 
         break
     }
-  }, [currentPlayerState])
+  }, [currentPlayerState, session_id])
 
   React.useEffect(() => {
+    // only execute the contents of this effect if the machine is in the
+    // process of loading. Any other Player Machine state change should bail
+    // early.
+    if (currentPlayerState !== 'loading') return
```

Comment:
allowing this effect to be triggered whenever the player state changes to loading, this allows the player to load when transitioning from 'subscribing' to 'loading'.